.prettierignore

```diff
@@ -1,3 +1,6 @@
+# Ignore packages dist
+packages/*/dist
```

Comment:
This could be potentially upstreamed to Core

Reply:
https://github.com/facebook/react-native/pull/41826

Reply:
Upstream PR merged, does this PR close now?